presto-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/environment/singlenode-hdp3/hadoop-entrypoint.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -euo pipefail
+
+echo "[$(date)] $0: configuring hadoop services"
+
+cd /docker/presto-product-tests/conf/environment/single-hdp3/
```

Comment:
i guess you'd need to cd back to the original location.
while bash has pushd, popd for this, i prefer not to cd at all. imo it makes scripts easier to understand